coredb-operator/src/controller.rs

```diff
@@ -87,6 +88,12 @@ impl CoreDB {
         let name = self.name_any();
         let coredbs: Api<CoreDB> = Api::namespaced(client.clone(), &ns);
 
+        // reconcile service account, role, and role binding
+        reconcile_rbac(self, ctx.clone()).await.map_err(|e| {
+            error!("Error reconciling service account: {:?}", e);
+            Action::requeue(Duration::from_secs(10))
```

Comment:
Like just wait a short period when it's an expected error, but for unexpected, wait like 5 minutes

Reply:
I just followed the other patterns for requeue time.  I can set it to `300` if thats what we want to do.  @ChuckHend what are your thoughts?

Reply:
I think I understand what @sjmiller609 is suggesting. Are there expected errors here? Elsewhere, we re-queue at 10 seconds because we are just waiting for an postgres instance to start up, for example.

Reply:
There shouldn't be any errors unless there is an issue talking to the kubernetes api during the creation of the serviceaccount, role and rolebinding.